WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m

```diff
@@ -915,6 +916,8 @@ - (void)configurePostCell:(ReaderPostTableViewCell *)cell atIndexPath:(NSIndexPa
     cell.postView.shouldShowAttributionMenu = shouldShowAttributionMenu;
     cell.postView.canShowActionButtons = self.hasWPComAccount;
     cell.postView.shouldShowAttributionButton = self.hasWPComAccount;
+    
+    [self setReblogButtonVisibilityOfPostView:cell.postView];
```

Comment:
It might be a good optimization to check blog visibility once and store its value for future use rather than needing to create a new instance of `BlogService` every time a cell is configured.